jenkins/main.groovy

```diff
@@ -127,16 +128,63 @@ pipeline {
                     echo 'Updating Github PR status'
                     def status
                     def description
+                    def message
                     if (currentBuild.currentResult ==  "SUCCESS") {
                         status = "SUCCESS"
                         description = "All stages succeeded"
+                        message = "Build has passed"
                     } else {
                         status = "FAILURE"
                         description = "Some stages failed"
+                        message = "Build has failed"
                     }
                     githubNotify context: prStatusContext, status: status, description: description
+                    notifyBuild(currentBuild.currentResult)
                 }
             }
         }
     }
 }
+
+def notifyBuild(String buildStatus = 'STARTED') {
+    def buildColor = ''
+    def buildEmoji = ''
+
+    // build status of null means successful
+    buildStatus =  buildStatus ?: 'SUCCESSFUL'
+
+    // Override default values based on build status
+    switch (buildStatus) {
+        case 'STARTED':
+            buildColor = 'warning'
+            buildEmoji = ':open_mouth:'
+            break;
+        case 'SUCCESSFUL':
```

Comment:
I guess this should be `case 'SUCCESS':` based on the githubNotify logic checking for `SUCCESS` not `SUCCESSFUL`.